onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -406,6 +407,19 @@ QNNExecutionProvider::QNNExecutionProvider(const ProviderOptions& provider_optio
     }
   }
 
+  static const std::string HTP_VTCM_BACKUP_BUFFER_SHARING = "enable_vtcm_backup_buffer_sharing";
```

Comment:
>enable_vtcm_backup_buffer_sharing [](http://example.com/codeflow?start=61&length=33)

If this option is available for Qnn 2.35 afterwards only. We should report warning if user set this option and the QNN version in using is lower.